folium/map.py

```diff
@@ -480,3 +480,46 @@ def __init__(self, bounds, padding_top_left=None,
             padding_bottom_right=padding_bottom_right,
             padding=padding,
         )
+
+
+class CustomPane(MacroElement):
+    """
+    Creates a custom pane to hold map elements.
+
+    Behavior is as in https://leafletjs.com/examples/map-panes/
+
+    Parameters
+    ----------
+    name: string
+        Name of the custom pane. Other map elements can be added
+        to the pane by specifying the 'pane' kwarg when constructing
+        them.
+    z_index: int or string, default 625
+        The z-index that will be associated with the pane, and will
+        determine which map elements lie over/under it. The default
+        (625) corresponds to between markers and tooltips. Default
+        panes and z-indexes can be found at
+        https://github.com/Leaflet/Leaflet/blob/v1.0.0/dist/leaflet.css#L87
+    pointer_events: bool, default False
```

Comment:
What are the consequences of setting the default here to `False`?

Maybe good to add a small section in the example notebook on how this option can be used.

Reply:
For my particular use case (labels on top of choropleth), setting this to False ensures that the new layer won't mask mouse events from lower layers, e.g. if you want a tooltip to pop up when you hover over a choropleth.

Right now I'm struggling to imagine a case where you'd want this to be True! But

Reply:
Thanks for explaining, sounds like `False` is a good default. I agree on leaving it in.